docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,25 +12,130 @@ keywords:
 
 # Contribute to What's New or release notes
 
+This topic explains the decisions and actions associated with collecting, writing, and publishing What's New content.
+
+> **Note:** This topic is only relevant for internal Grafana Labs contributors.
+
+## What's New doc development process
+
+Developing directly in Markdown reduces errors and removes a manual (toilsome and error-prone step), by creating, editing, discussing, and publishing in the same format from which we will publish in GitHub, using the Grafana repo. 
+
+Consider adding an image, though be aware that we will not maintain them, as the document reflect a "point in time." If you need to add an image, refer to [Image, diagram, and screenshot guidelines]({{< relref "../image-guidelines/" >}}).
+
+### When the prior release is complete
+
+Complete the following steps within one week after the prior release goes out.
+
+1. The Docs Squad identifies the technical writer responsible for overseeing the What’s New content development process for the upcoming release.
+1. The identified technical writer cuts a branch and creates a PR with an empty What's New doc to be populated with the What’s New content for the next release. 
```

Comment:
I know we're still hammering out the process, but should we not include the Upgrade Guide here too? I'm not sure how granular we wanna be here, but it seems like if we're not going to document this anywhere else, we should get nitty gritty about the way this PR should be set up.
```suggestion
1. The identified technical writer cuts a branch and creates a draft PR with an empty What's New doc to be populated with the What’s New content for the next release along with the updated What's New index page. This PR should also include the new Upgrade Guide page (process TBA). This PR should not have a `backport` label initially.
```